dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java

```diff
@@ -289,6 +289,60 @@ public void testSingleFileMove() throws IOException, ExecutionException, Interru
     }
   }
 
+  @Test
+  public void testSingleFileCopySkip() throws IOException, ExecutionException,
```

Comment:
do you mind adding some tests regarding copying a folder with different scenarios, thanks!